knowage-vue/src/i18n/en_US.json

```diff
@@ -127,6 +128,43 @@
             "hint": "The metadata allows you to access and write metadata inside your users document. It's possible to change the type between the allowed ones and to name and describe it. Click on one existing item from the list on the side, or press the \"+\" button to add one.",
             "title": "Metadata Management"
         },
+        "usersManagement": {
+            "title": "Users list",
+            "userId": "User ID",
+            "fullName": "Full Name",
+            "validation": {
+                "sameAsPassword": "Username must be the same as password",
+                "extendedAlphanumeric": "Invalid format",
+                "maximumCharacters": "The maximum length allowed is {length}",
+                "required": "{field} is required"
+            },
+            "detail": "Detail",
+            "roles": "Roles",
+            "attributes": "Attributes",
+            "unlockUser": "Unlock user",
+            "unsavedChangesHeader": "Unsaved changes",
+            "unsavedChangesMessage": "You have not saved your recent changes. Continue without saving ?",
+            "blockedUserInfo": "This user reached the maximum number of failed login attempts and therefore he was locked. Do you want to unlock him?",
+            "defaultRoleInfo": "Select a role if you want to set it as the default. If no role is selected, it will be possible switch between roles from the Knowage Administration menu",
+            "valueDescription": "Value description",
+            "confirmDeleteMessage": "Are you sure you want to delete the selected {item}?",
```

Comment:
This label should be common in most list interfaces, move to a common.confirm section